elements/asq-java-q-presenter.html

```diff
@@ -0,0 +1,274 @@
+            words = text.text().trim().split(/\s+/).reverse(),
+            word,
+            line = [],
+            lineNumber = 0,
+            lineHeight = 1.1, // ems
+            y = text.attr("y")  || 0,
+            dy = parseFloat(text.attr("dy")) || 0,
+            tspan = text.text(null).append("tspan")
+              .attr("x", 0)
+              .attr("y", y)
+              .attr("dy", dy + "em");
+        while (word = words.pop()) {
+          line.push(word);
+          tspan.text(line.join(" "));
+          if (tspan.node().getComputedTextLength() > width) {
+            line.pop();
+            tspan.text(line.join(" "));
+            line = [word];
+            tspan = text.append("tspan").attr("x", 0).attr("y", y).attr("dy", ++lineNumber * lineHeight + dy + "em").text(word);
+          }
+          // align center
+          var w = tspan.node().getComputedTextLength();
+          var tspanX = (parseFloat(width) / 2) - (w / 2);
+          tspan.attr("x", tspanX)
+        }
+      });
+    }
+
+    Polymer({
+      is: 'asq-java-q-presenter',
+
+      properties: {
+        _answers: {
+          type: Array,
+          notify: true,
+          value: function() { return []; }
+        },
+        /**
+         * Set to true to style the element as disabled.
+         * Binded to `readonly` of ace-editor.
+         */
+        disabled: {
+          type: Boolean,
+          value: false,
+          notify: true,
+          reflectToAttribute: true
+        },
+        /**
+         * The fontSize property of the ace-element.
+         */
+        fontSize: {
+          type: String,
+          value: "12px",
+          notify: true,
+          reflectToAttribute: true
+        },
+        /**
+         * The mode property of the ace-element.
+         */
+        mode: {
+          type: String,
+          value: 'java',
+          reflectToAttribute: true,
+          notify: true
+        },
+        /**
+         * The tabSize property of the ace-element.
+         */
+        tabSize: {
+          type: Number,
+          value: 2,
+          notify: true,
+          reflectToAttribute: true,
+        },
+        /**
+         * The theme property of the ace-element.
+         */
+        theme: {
+          type: String,
+          value: 'monokai',
+          notify: true,
+          reflectToAttribute: true,
+        },
+        /**
+         * The value property of the ace-element.
+         */
+        value: {
+          type: String,
+          value: '',
+          notify: true,
+          reflectToAttribute: true,
+          observer: '_valueChanged'
+        },
+        /**
+         * The wrap property of the ace-element.
+         */
+        wrap: {
+          type: Boolean,
+          value: false,
+          notify: true,
+          reflectToAttribute: true,
+        },
+        _selectedSubmissionItem: {
+          observer: '_selectedSubmissionItemChanged'
+        },
+        _selectedTab:{
```

Comment:
It's used by the presenter view to show a submission when clicked. You can merge the PR 👍